sources/packages/backend/apps/api/src/route-controllers/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from "./user/user.controller";
-export * from "./program-year/program-year.controller";
+export * from "./program-year/program-year.student.controller";
```

Comment:
it should be plural  `students`. you can also refer other controllers, 
![image](https://user-images.githubusercontent.com/77353155/208193120-9ad6ef54-8851-4ccf-bb21-8c5e77f84d10.png)